modules/statics/src/account.ts

```diff
@@ -982,3 +1014,63 @@ export function tavaxErc20(
     primaryKeyCurve
   );
 }
+
+/**
+ * Factory function for FIAT currency instances.
+ *
+ * @param name unique identifier of the currency
+ * @param fullName Complete human-readable name of the currency
+ * @param asset Asset which this coin represents. This is the same for both mainnet and testnet variants of a coin.
+ * @param decimalPlaces Number of decimal places this coin supports (divisibility exponent)
+ * @param network Network object for this coin
+ * @param prefix? Optional coin prefix. Defaults to empty string
+ * @param suffix? Optional coin suffix. Defaults to coin name.
+ * @param features? Features of this coin. Defaults to the DEFAULT_FEATURES defined in `AccountCoin`
+ * @param isToken? Whether or not this account coin is a token of another coin
+ * @param primaryKeyCurve The elliptic curve for this chain/token
+ */
+export function fiatToken(
+  name: string,
+  fullName: string,
+  asset: UnderlyingAsset,
+  decimalPlaces: number,
+  network: BaseNetwork,
+  prefix = '',
+  suffix: string = name.toUpperCase(),
+  features: CoinFeature[] = FiatCoin.DEFAULT_FEATURES,
+  isToken = true,
+  primaryKeyCurve: KeyCurve = KeyCurve.Secp256k1
+) {
+  return Object.freeze(
+    new FiatCoin({
+      name,
+      fullName,
+      network,
+      prefix,
+      suffix,
+      features,
+      decimalPlaces,
+      isToken,
+      asset,
+      primaryKeyCurve,
+    })
+  );
+}
+
+/**
+ * Factory function for testnet FIAT currency instances.
+ *
+ * @param name unique identifier of the currency
+ * @param fullName Complete human-readable name of the currency
+ * @param asset Asset which this coin represents. This is the same for both mainnet and testnet variants of a currency.
+ * @param decimalPlaces Number of decimal places this currency supports (divisibility exponent)
+ */
+export function tfiatToken(
```

Comment:
Are we missing a `fiatToken()` function that is the main net equivalent? Or is that to come later?

Reply:
That comes later, addressed with: https://bitgoinc.atlassian.net/browse/STLX-13365